src/components/packages/Package-writing.svelte

```diff
@@ -0,0 +1,37 @@
+<style>
+    .text{
+        display: flex;
+        flex-direction: column;
+        width: 60%;
```

Comment:
Gets a little cramped on mobile, I'd bump up this percent when things get narrow.